openmc/trigger.py

```diff
@@ -96,6 +97,20 @@ def add_score(self, score):
                   'it is not a string'.format(score)
             raise ValueError(msg)
 
+        # If this is a total/flux/scattering moment, use generic moment order
+        regexp = re.compile(r'-[0-9]')
+        if regexp.search(score) is not None:
+            score = score.strip(regexp.findall(score)[0])
+            score += '-n'
+        regexp = re.compile(r'-[p|P][0-9]')
+        if regexp.search(score) is not None:
+            score = score.strip(regexp.findall(score)[0])
+            score += '-pn'
+        regexp = re.compile(r'-[y|Y][0-9]')
```

Comment:
You could probably combine these three separate cases into a single regex using `|`